apps/builder/app/builder/features/style-panel/style-layer-utils.test.ts

```diff
@@ -360,3 +362,180 @@ describe("boxShadowUtils", () => {
     `);
   });
 });
+
+test("Generates humane layer names for shadow style layer", () => {
+  expect(
+    getHumanizedTextFromLayer(
+      "boxShadow",
+      (
+        parseShadow(
+          "boxShadow",
+          "inset 2px 2px 5px rgba(0, 0, 0, 0.5)"
+        ) as LayersValue
+      ).value[0] as TupleValue
+    )
+  ).toMatchInlineSnapshot(`
+{
+  "color": {
+    "a": 0.5,
+    "b": 0,
+    "g": 0,
+    "r": 0,
+  },
+  "name": "Inner shadow:  2px 2px 5px",
+  "value": "inset 2px 2px 5px rgba(0, 0, 0, 0.5)",
+}
+`);
+
+  expect(
+    getHumanizedTextFromLayer(
+      "boxShadow",
+      (
+        parseShadow(
+          "boxShadow",
+          "3px 3px 10px rgba(255, 0, 0, 0.7)"
+        ) as LayersValue
+      ).value[0] as TupleValue
+    )
+  ).toMatchInlineSnapshot(`
+{
+  "color": {
+    "a": 0.7,
+    "b": 0,
+    "g": 0,
+    "r": 255,
+  },
+  "name": "Outer shadow:  3px 3px 10px",
```

Comment:
Why is it "Outer shadow" vs "Drop Shadow" both both upper case? or "Inner shadow"

Which one should be used consistently?